### DISCUSSION TITLE
Webview and Webview_QT build dependencies

### DISCUSSION THREAD

**Discussion by ruben-herold** [FreeRDP/FreeRDP#10965] [Q&A]
hi,

I try to rebuild the debian freerdp3 packages with Webview. What are the build dependencies for it?

**Comment by akallabeth** [comment]
@ruben-herold for the `gtk` version check the nightly package debian stuff in the repo, for qt, well the whole of `QWebView` and dependencies

**Comment by ruben-herold** [comment]
@akallabeth  I don't see in the nightly package something with gtk or webview. 

I have  the task to find a configuration for users to connect via rdp-gateway with local auth to a system with aad auth. 
My current try is:
`
xfreerdp3  /sec:aad /ipv4 /redirect-prefer:IP /dynamic-resolution /cert:ignore /gateway:g:192.168.14.53,u:gwuser01 /v:win11-aad.test.de   '/u:testuser@aaddomain.de'

`
It works, but then the user must copy a URL to a web browser and so on. This is much too complicated for the users.

**Reply by akallabeth** [reply]
https://github.com/FreeRDP/FreeRDP/blob/master/packaging/deb/freerdp-nightly/control lists all dependencies for the gtk version.
for the `Qt` version you also need `qtwebengine5-dev`

if still unsure, all compile/link related stuff is here: https://github.com/FreeRDP/FreeRDP/blob/master/client/SDL/common/aad/CMakeLists.txt

[note] we´ve deactivated this as it has a few drawbacks, but for your installation that might work. (best test which one works best for you, both have a few bugs we did not have time to iron out)
